app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
   has_one_attached :photo
+  has_many :classrooms_as_student, through: :attendances, source: :classrooms
+  has_many :classrooms
```

Comment:
double associations on classroom?

Reply:
![image](https://github.com/user-attachments/assets/1e3fad72-4452-4f7e-befe-583247d226a3)
One would be the connection between user and classrooms as a teacher and the other is as student through attendance.